### PR TITLE
Refs #35058 -- Added support for measured geometries to GDAL LineString.

### PR DESCRIPTION
**Pull request by smithdc1**

### CODE REVIEW

File: django/contrib/gis/gdal/prototypes/geom.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def topology_func(f):
 get_geom_type = int_output(lgdal.OGR_G_GetGeometryType, [c_void_p])
 get_point_count = int_output(lgdal.OGR_G_GetPointCount, [c_void_p])
 get_point = void_output(
-    lgdal.OGR_G_GetPoint,
+    lgdal.OGR_G_GetPointZM,
     [c_void_p, c_int, POINTER(c_double), POINTER(c_double), POINTER(c_double)],
```
Comment (ngnpope) [inline django/contrib/gis/gdal/prototypes/geom.py:141]: I think this should be updated?

```suggestion
    [
        c_void_p, c_int, POINTER(c_double), POINTER(c_double), POINTER(c_double),
        POINTER(c_double),
    ],
```

See the [documentation](https://gdal.org/api/vector_c_api.html#_CPPv416OGR_G_GetPointZM12OGRGeometryHiPdPdPdPd).

File: docs/releases/5.1.txt [hunk 2]
```diff
@@ -78,10 +78,12 @@ Minor features
 * The new :meth:`.OGRGeometry.set_3d` method allows addition and removal of the
   ``Z`` coordinate dimension.
 
-* :class:`~django.contrib.gis.gdal.OGRGeometry` and
-  :class:`~django.contrib.gis.gdal.Point` now support measured geometries
-  via the new :attr:`.OGRGeometry.is_measured` and :attr:`.Point.m` properties,
-  and the :meth:`.OGRGeometry.set_measured` method.
+* :class:`~django.contrib.gis.gdal.OGRGeometry`,
+  :class:`~django.contrib.gis.gdal.Point` and
```
Comment (ngnpope) [inline docs/releases/5.1.txt:82]:
```suggestion
  :class:`~django.contrib.gis.gdal.Point`, and
```

File: docs/releases/5.1.txt [hunk 3]
```diff
@@ -78,10 +78,12 @@ Minor features
...
+  :class:`~django.contrib.gis.gdal.LineString` now support measured geometries
+  via the new :attr:`.OGRGeometry.is_measured`, :attr:`.Point.m` and
```
Comment (ngnpope) [inline docs/releases/5.1.txt:84]:
```suggestion
  via the new :attr:`.OGRGeometry.is_measured`, :attr:`.Point.m`, and
```

**Review by ngnpope — CHANGES_REQUESTED** [review]
Looking good - one oversight a few small nits.

File: django/contrib/gis/gdal/prototypes/geom.py (hunk 1 above)
Comment (smithdc1) [reply django/contrib/gis/gdal/prototypes/geom.py:141]: Yes. Quite.

Need to work out how/why the tests passed.

File: docs/releases/5.1.txt (hunk 3 above)
Comment (smithdc1) [reply docs/releases/5.1.txt:84]: Ah yes, I forgot about the oxford comma preference. However, I'm not sure about this one. 

We've two groups, a list of properties and a method. Currently there's a comma before the 'and' that joins these two groups (line 85). Adding another one here makes the distinction between the two groups less clear (for me).

File: django/contrib/gis/gdal/prototypes/geom.py (hunk 1 above)
Comment (smithdc1) [reply django/contrib/gis/gdal/prototypes/geom.py]: I'm not sure why the the tests passed with an incorrect signature. While the python [docs](https://docs.python.org/3/library/ctypes.html#ctypes._FuncPtr.argtypes) have the following comment, I'm not entirely sure if that is the scenario here.

> functions using the C calling convention accept additional, unspecified arguments as well.

If we're happy the signature is now correct and the tests pass, is that enough?

Comment (felixxm) [reply django/contrib/gis/gdal/prototypes/geom.py]: I can live with that.

File: docs/releases/5.1.txt (hunk 3 above)
Comment (felixxm) [reply docs/releases/5.1.txt]: I think it's clearer without references.

**Review by felixxm — COMMENTED** [review]
@smithdc1 Thanks :+1: I pushed tiny edits to the release note.

**Pull request merged**